install/installer/pkg/common/common.go

```diff
@@ -505,6 +505,7 @@ func KubeRBACProxyContainerWithConfig(ctx *RenderContext) *corev1.Container {
 			"--logtostderr",
 			fmt.Sprintf("--insecure-listen-address=[$(IP)]:%d", baseserver.BuiltinMetricsPort),
 			fmt.Sprintf("--upstream=http://127.0.0.1:%d/", baseserver.BuiltinMetricsPort),
+			"--http2-disable",
```

Comment:
Why disable it?

Reply:
The last two releases are related to CVEs that exploit HTTP/2. Also, we don't need/use any HTTP/2 feature.